### DISCUSSION TITLE
Call for contributors to write the paper

### DISCUSSION THREAD

**Discussion by ajshajib** [AstroBridge/BDLensing#75] [Announcements]
I have created the issues on writing different sections in the paper: #70, #71, #72, #73, and #74. 

If you would like to volunteer to write any or multiple of these sections, **please assign the issue(s) to yourself.** It is welcome to have multiple assignees for one section. 

Here is the Overleaf link with edit access: https://www.overleaf.com/5822996548fqxvtfjtzszk.

As an example paper to follow on the writing, [this paper](https://arxiv.org/abs/1807.09278) will be most common in writing up the Data and Method sections and most likely also for the Introduction.

[This is the HST program](https://archive.stsci.edu/proposal_search.php?id=15867&mission=hst ), where our data comes from.

**Comment by ajshajib** [comment]
A note on **plagiarism**: I ask that you please be careful not to plagiarise previous papers/materials/books. It is a big no-no to plagiarise in scientific papers; even self-plagiarism is not OK. However, if it is needed to convey the same or similar information, it is fine to **paraphrase** from previous materials.

arXiv and journals use plagiarism checkers, and they can easily catch copy-pasting. For example, arXiv will add a note in the "Comments" for this paper that got caught plagiarising: https://arxiv.org/abs/1904.12594.

**Comment by ajshajib** [comment]
We plan to submit to A&A, so we will follow their policies on writing. 

Please read [this guideline](https://www.aanda.org/for-authors/author-information/paper-organization#about_language) (especially "About the language" and "Paper organization") and [this guideline](https://www.aanda.org/for-authors/language-editing/1-introduction). Please pay specific attention to "6. Verb tense" section. We want to strictly follow the guidelines on language editing (the second guideline above), so please take special care to follow them in your writing. Otherwise, it will create a lot of work for me to edit and improve the writing.

@rafee-adnan @aaimtiaz, please tag everyone else who is directly contributing to writing the draft to make them notice these links.

**Reply by ajshajib** [reply]
We will use US spelling conventions.

**Reply by rafee-adnan** [reply]
@JobairHasan @Rajit13 @SulymanHossain @ZareefJafar @mamun-nahid

**Reply by ajshajib** [reply]
On writing tasks, we had a deadline of Nov 12 for some text to be added to Overleaf for Sections 1-4. I see that only I have adhered to this deadline. While using Google Docs is fine to create some initial text, but they need to be transferred to the Overleaf draft in a timely fashion, so that I can track the progress on writing. Also, **I would suggest getting yourself familiar with Overleaf and working directly on there.** One goal of this project is for you to be familiar with the tools physicists use in their research. I have never seen anyone in astronomy and physics write a paper on Docs or Word. So, learning to use Overleaf (or any latex editor in general) will improve your research productivity in the long run.

**Comment by ajshajib** [comment]
A guide on writing papers, by Prof. Anna Sajina of Tufts University: http://cosmos2.phy.tufts.edu/~asajina/writing_guide.pdf

Here's another guide: https://arxiv.org/abs/2110.05503
